src/screens/Settings/General/ThemeSettingsRow.js

```diff
@@ -13,46 +13,49 @@ import { setTheme } from "../../../actions/settings";
 import Check from "../../../icons/Check";
 
 export default function ThemeSettingsRow() {
-  const theme = useSelector(themeSelector);
+  const { t } = useTranslation();
+  const currentTheme = useSelector(themeSelector);
   const { colors } = useTheme();
   const dispatch = useDispatch();
   const [isOpen, setIsOpen] = useState(false);
-  const selectTheme = t => () => {
-    dispatch(setTheme(t));
+  const selectTheme = theme => () => {
+    dispatch(setTheme(theme));
   };
 
   const onClose = useCallback(() => setIsOpen(false), []);
 
   return (
     <>
       <SettingsRow
-        title={<Trans i18nKey="settings.display.theme" />}
-        desc={<Trans i18nKey="settings.display.themeDesc" />}
+        title={t("settings.display.theme")}
+        desc={t("settings.display.themeDesc")}
         arrowRight
         onPress={() => setIsOpen(true)}
         alignedTop
       >
         <LText semiBold color="grey">
-          <Trans i18nKey={`settings.display.themes.${theme}`} />
+          {t(`settings.display.themes.${currentTheme}`)}
         </LText>
       </SettingsRow>
       <BottomModal isOpened={isOpen} onClose={onClose}>
         <View style={styles.modal}>
-          {["light", "dusk", "dark"].map((t, i) => (
+          {["light", "dusk", "dark"].map((theme, i) => (
             <Touchable
               event="ThemeSettingsRow"
-              eventProperties={{ theme: t }}
-              key={t + i}
-              onPress={selectTheme(t)}
+              eventProperties={{ theme }}
+              key={theme + i}
+              onPress={selectTheme(theme)}
               style={[styles.button]}
             >
               <LText
-                {...(theme === t ? { semiBold: true } : {})}
+                {...(currentTheme === theme ? { semiBold: true } : {})}
                 style={[styles.buttonLabel]}
               >
-                {t}
+                {t(`settings.display.themes.${theme}`)}{" "}
```

Comment:
Just commenting in case `{" "}` is a typo

Reply:
Yep not needed, don't know how this got there, might be editor's auto-format going wild 🤔

Reply:
It's prettier/eslint adding it randomly sometimes in JSX, don't know when exactly :/